src/views/FindPark.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <section>
+    <input
+      class="search-input"
+      type="text"
+      placeholder="Search by name, city or zip code"
+      v-model="searchTerm"
+    />
+    <button class="search-button" @click="search('searchTerm')">
+      Get Started - woof!
+    </button>
+    <p v-if="searchTerm === ''">Please enter a park name, city, or zip code</p>
+  </section>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      searchTerm: '',
+    };
+  },
+  methods: {
+    search() {
+      if (this.searchTerm !== '') {
+        console.log(this.searchTerm);
+        this.searchTerm = '';
+        //functionality for search goes here...
```

Comment:
Thanks for making this comment. It's helpful To know where we need to add the functionality

Reply:
Thanks! Added `lang="scss"` to my file!